%\VignetteIndexEntry{getPathHelperFiles}
%\VignetteDepends{getPathHelperFiles}
%\VignettePackage{getPathHelperFiles}
%\VignetteEngine{utils::Sweave}

\documentclass{article}

<<style-Sweave, eval=TRUE, echo=FALSE, results=tex>>=
BiocStyle::latex()
@

\newcommand{\exitem}[3]{%
  \item \texttt{\textbackslash#1\{#2\}} #3 \csname#1\endcsname{#2}.%
}

\title{\bf CopyhelpeR: Helper files for CopywriteR.}
\author{Thomas Kuilman}

\begin{document}

\maketitle

\begin{center}
  Department of Molecular Oncology \\*
  Netherlands Cancer Institute \\*
  The Netherlands \\*
\end{center}

\begin{center}
  \email{t.kuilman@nki.nl} or \email{thomaskuilman@yahoo.com}
\end{center}

\tableofcontents

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Usage}

To obtain the full path to the the helper files folder for the relevant
reference genomes (hg19, mm9 or mm10), use the
\Rcode{getPathHelperFile()} function:

<<getPathBam, print=FALSE, eval=TRUE>>=
library(CopyhelpeR)
getPathHelperFiles("hg19")
@

The helper files are provided as annotation data for the \Biocpkg{CopywriteR}
package, which uses off-target reads from targeted sequencing for copy number
detection. The helper files contain pre-assembled 1kb bin GC-content and
mappability bed files for the reference genomes hg19, mm9 and mm10. In addition,
it contains a blacklist filter to remove regions that display CNV.

\end{document}